### PR TITLE
Issue #614: Suppress UnusedProperty IDEA violations

### PR DESCRIPTION
**Pull request by Vyom-Yadav**
#614 
Suppression was generated by IDEA

### CODE REVIEW

**Comment by romani** [comment]
@nick-mancuso , why we have two IDEA inspection executions ?
image

File: releasenotes-builder/config/checkstyle.properties [hunk 1]
```diff
@@ -1,3 +1,5 @@
+# suppress inspection "UnusedProperty" for whole file
+# properties are used at "misc/build.xml"
```
Comment (romani) [inline releasenotes-builder/config/checkstyle.properties:2]: how comments change anything ?

**Review by romani — CHANGES_REQUESTED** [review]
items:

File: releasenotes-builder/config/checkstyle.properties (hunk 1 above)
Comment (Vyom-Yadav) [reply releasenotes-builder/config/checkstyle.properties:2]: That is not a regular comment, that is a suppression comment.

**Review by romani — APPROVED** [review]
Ok to merge

**Comment by nick-mancuso** [comment]
@Vyom-Yadav please resolve conflicts

**Comment by Vyom-Yadav** [comment]
> @Vyom-Yadav please resolve conflicts

Done.

**Review by nick-mancuso — APPROVED** [review]

**Pull request merged**